onnxruntime/core/session/environment.cc

```diff
@@ -264,13 +265,16 @@ Status Environment::Initialize(std::unique_ptr<logging::LoggingManager> logging_
       // preserve this order until <training schemas>: this depends on operatorsetschema registration.
       training::RegisterTrainingOpSchemas();
 #endif
+#ifdef ENABLE_TRAINING_CORE
+      // <training schemas>
+      // This can also be moved inside enable_training. Needs more investigation
+      training::GraphTransformerRegistry::GetInstance().RegisterExternalGraphTransformers();
```

Comment:
if i remember correctly, this api is added for Apollo usage, which we don't need it anymore. we can double confirm whether we can remove it, but i believe you don't need it for on device training.

Reply:
makes sense will cover this in a separate PR which I am working on right now... Will merge this PR.